extension/src/Runner/testRunnerContext.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import { TestSuite } from "../protocols";
+
+export interface ITestRunnerContext {

```

Comment:
This class is actually the parameters for ITestRunner.run. Suggest:
Eliminate it and expose the class members as parameters of ITestRunner.run
or
Rename the class to ITestRunnerParameters.
I would suggest the former because it's more explicit contract-wise.

Reply:
we do have a lot of context info of this run, like `transactionid`, `storagepath`, `port`, etc. Putting them directly as the parameters of the method would make the interface vulnerable to change. However, I agree that we could put `tests` and `isDebugMode` as parameters directly